app/controllers/application_controller.rb

```diff
@@ -1,2 +1,5 @@
 class ApplicationController < ActionController::Base
+  def current_user
```

Comment:
- [ ] Kindly allow user to create comment on behalf on the `current_user` as per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/forms_project_v1_1.md#:~:text=Create%20a%20comment%20on%20behalf%20of%20the%20current_user%20(use%20the%20method%20that%20you%20created%20in%20your%20ApplicationController ).

Reply:
thank you, I add it